### DISCUSSION TITLE
benchmark specific files

### DISCUSSION THREAD

**Discussion by pedrospeixoto** [pedrospeixoto/MPAS-PXT#9] [General]
Hi,

I think we should keep benchmark specific files in folders, not in main. Also, things that are very specific for a benchmark should be in branch no in master. For instance:
target_domain.Catarina
could be in default_input, but maybe not in main folder. Or it could be just in branch.

@daniloceano is that ok with you? We need to clean up mpas-br as new users are coming.

**Comment by favba** [comment]
Hi Guys,

I went ahead and removed the files from the main folder.
They are already in the `default_inputs` folder.

@daniloceano, if the files are absolutely needed to be in the main folder we can easily revert that change.
Altough I'm guessing those files were just added by mistake together with other real changes that you wanted to push...
